pkg/textparse/openmetricsparse.go

```diff
@@ -25,10 +27,13 @@ import (
 
 	"github.com/pkg/errors"
 
+	"github.com/prometheus/prometheus/pkg/exemplar"
 	"github.com/prometheus/prometheus/pkg/labels"
 	"github.com/prometheus/prometheus/pkg/value"
 )
 
+var allowedSuffixes = []string{"_total", "_bucket"}
```

Comment:
Perhaps make these `[][]byte{[]byte("_total"), []byte("_bucket")}` so that don't need to do a `[]byte(strValue)` in the validate code later?